### DISCUSSION TITLE
Directii unde TESTERII pot fi asignati

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#128] [HEAD OF TESTING]
**1. website-ul peviitor.ro - frontend - repo: search-engine**
2. website-ul peviitor.ro - backend - repo: api
**3. validator admin.peviitor.ro - UI** 
**4. validator admin.peviitor.ro - Date** (backlog separat pe firmele care le are/gardian)
**5. peviitor.ro as a user - locurile de munca --- puna bug de pe butonul icon footer github**
6. echipele de scraperisti (Iurie, Cristi Olteanu, Rares, Laurentiu, Andrei, Sebi)
**7. oportunitatisicariere.ro - website (Strategy, TC, bugs)**
8. orase.peviitor.ro - localitati
9. orase-ui -- unificat cu orase si UI-ul sa fie in acelasi REPO cu orase.peviitor.ro plus ne trebe alt DESIGN + ne trebuie HARTI pe UI
10. infokiosk -- strategy, test cases, requirements

**Comment by AdelinaGuliman** [comment]
Echipa de testare pentru Repo: Serch engine
1. Adelina Guliman
2. Ionut Pintilie
3. Maxima 
4. Razvan Gradea
5. Daniela Popa

**Comment by AdelinaGuliman** [comment]
Echipa de testare pentru Repo: Validator UI
1. Diana Dragoi
2. Cosmin
3. Elena Istrate 
4. Lucian
5. Marius Cimpanu
6. Simona Nemtanu

**Comment by AdelinaGuliman** [comment]
Echipa de testare pentru Repo: Validator Date
1. Catalina Ion 
2. Ionela Anton
3. Alexandru Cantacuz

**Comment by AdelinaGuliman** [comment]
Echipa de testare pentru Repo: oportunitati si cariere
1. Florin Nistor
2. Bianca Jianu
3. Bogdan Dumitrache
4. Diana Dragoi
5. Georgiana Redenstein
6. Camelia Melinte

**Comment by AdelinaGuliman** [comment]
Echipa de testare As a user
1. Carmen Steliana (Android, IOS)
2. Elisa (IOS, Browser: Safari)
4. Cati (PC, Browser: Opera)
5. Alin (LabView) (PC, Browser: Opera)
6. Cosmin (PC, Browser: Mozilla)
7. Ana-Maria Talmacel (PC, Browser: Chrome)
